modules/android_demos/coco_detection_android_demo/app/src/main/java/org/intel/openvino/MainActivity.java

```diff
@@ -0,0 +1,364 @@
+                    InputStream inputStream = getApplicationContext().getAssets().open(fileName);
+                    OutputStream outputStream = new FileOutputStream(outputFilePath);
+                    byte[] buffer = new byte[5120];
+                    int length = inputStream.read(buffer);
+                    while (length > 0) {
+                        outputStream.write(buffer, 0, length);
+                        length = inputStream.read(buffer);
+                    }
+                    outputStream.flush();
+                    outputStream.close();
+                    inputStream.close();
+                } catch (Exception e) {
+                    Log.e("CopyError", "Copying model has failed.");
+                    System.exit(1);
+                }
+            }
+        }
+    }
+
+    private void createRandomColor(int length) {
+        Random r = new Random(999);
+        randomColor = new Scalar[length];
+
+        for (int i = 0; i < length; i++) {
+            randomColor[i] =
+                    new Scalar(r.nextDouble() * 255, r.nextDouble() * 255, r.nextDouble() * 255);
+        }
+    }
+
+    private void processNetwork() {
+        // Set up camera listener
+        mOpenCvCameraView = (CameraBridgeViewBase) findViewById(R.id.CameraView);
+        mOpenCvCameraView.setVisibility(CameraBridgeViewBase.VISIBLE);
+        mOpenCvCameraView.setCvCameraViewListener(this);
+        mOpenCvCameraView.setCameraPermissionGranted();
+        mOpenCvCameraView.enableFpsMeter();
+        mOpenCvCameraView.setMaxFrameSize(320, 240);
+
+        // Initialize model
+        copyFiles();
+        Core core = new Core(modelDir + "/" + PLUGINS_XML);
+        Model net = core.read_model(modelDir + "/" + MODEL_XML);
+        Log.i(APPTAG, "load ok...");
+
+        // Set config of the network
+        PrePostProcessor p = new PrePostProcessor(net);
+        p.input().tensor().set_element_type(ElementType.u8).set_layout(new Layout("NHWC"));
+
+        p.input().preprocess().resize(ResizeAlgorithm.RESIZE_LINEAR);
+        p.input().model().set_layout(new Layout("NCHW"));
+        p.build();
+
+        CompiledModel compiledModel = core.compile_model(net, DEVICE_NAME);
+        inferRequest = compiledModel.create_infer_request();
+
+        // System info
+        ActivityManager activityManager = (ActivityManager) getSystemService(ACTIVITY_SERVICE);
+        ActivityManager.MemoryInfo info = new ActivityManager.MemoryInfo();
+        activityManager.getMemoryInfo(info);
+        Log.i(APPTAG, "residue memory : " + (info.availMem >> 20) + "M");
+
+        tm = new TickMeter();
+        createRandomColor(COCO_CLASSES_91.length);
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        try {
+            System.loadLibrary(OPENCV_LIBRARY_NAME);
+            System.loadLibrary(Core.NATIVE_LIBRARY_NAME);
```

Comment:
Now we're loading native OV libs in OV Java bindings, so we don't need to do it here:
https://github.com/openvinotoolkit/openvino_contrib/blob/7239f8201bf18d953298966afd9161cff50b2d38/modules/java_api/src/main/java/org/intel/openvino/NativeLibrary.java#L16